v2/dtos/deviceservice.go

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package dtos
+
+// DeviceService represents a service that is responsible for proxying connectivity between a set of devices and the
+// EdgeX Foundry core services.
+// This object and its properties correspond to the DeviceService object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-metadata/2.x#/DeviceService
+type DeviceService struct {
+	Id             string   `json:"id,omitempty"`
+	Name           string   `json:"name" validate:"required"`
```

Comment:
Should we also be using validation tag to ensure name is formatted properly? i.e. do we restrict the characters that can be in the name? i.e. is space valid in the name. This should be consistent across on Name properties. Looks like V1 doesn't restrict contents of Name, but should we in V2? (@cloudxxx8 , @tonyespy , @jpwhitemn )